elm.json

```diff
@@ -6,7 +6,7 @@
     "elm-version": "0.19.1",
     "dependencies": {
         "direct": {
-            "FMFI-UK-1-AIN-412/elm-formula": "2.0.0",
+			"FMFI-UK-1-AIN-412/elm-formula": "3.1.1",
```

Comment:
Set up your editor to indent using spaces only. These are apparently tabs set to a non-standard width (≠8). They break alignment in different editors.

Reply:
Would you fix this, @MatejKomlossy, pls?